lightning-invoice/src/payment.rs

```diff
@@ -15,9 +15,10 @@
 //! and payee using information provided by the payer and from the payee's [`Invoice`], when
 //! applicable.
 //!
-//! [`InvoicePayer`] is parameterized by a [`LockableScore`], which it uses for scoring failed and
-//! successful payment paths upon receiving [`Event::PaymentPathFailed`] and
-//! [`Event::PaymentPathSuccessful`] events, respectively.
+//! [`InvoicePayer`] uses its [`Router`] parameterization for optionally notifying scorers on
+//! the failure and success of payment paths upon receiving the  [`Event::PaymentPathFailed`] and
+//! [`Event::PaymentPathSuccessful`] events, respectively. It also does the same for payment probe
+//! failure and success events using [`Event::ProbeFailed`] and [`Event::ProbeSuccessful`].
```

Comment:
Similarly here.

Reply:
Addressed in c5206b3bf646b61db957ef7dd80b79538114181d